src/firefly/js/visualize/task/PlotHipsTask.js

```diff
@@ -304,13 +304,14 @@ async function makeHiPSPlot(rawAction, dispatcher) {
 
 
 
-export function createHiPSMocLayerFromPreloadedTable({tbl_id,title, fitsPath, mocUrl, plotId, visible=false, color, mocGroupDefColorId} ) {
+export function createHiPSMocLayerFromPreloadedTable({tbl_id,title, fitsPath, mocUrl, plotId, visible=false,
+                                                         color, mocGroupDefColorId, attachAllPlot=false} ) {
     const table= getTblById(tbl_id);
```

Comment:
`attachAllPlot` is added to createHiPSMocLayerFromPreloadedTable, but, I don't see where it's used in your PR.  This is normal?

Reply:
It is only called from one place and I made the default false. If used again (I think I probably will) there is a case which a it could pass `true` I made it a parameter instead of hardcoding it on line 314